### DISCUSSION TITLE
Hi! is there a way to get current standings?

### DISCUSSION THREAD

**Discussion by danielsaed** [theOehrly/Fast-F1#215] [Q&A] [answered]
I was wondering if there is a way to get the drivers and teams standings, i didn't find anything related to it in the documentation, but maybe I'm missing something, thanks!

**Comment by theOehrly** [comment, marked as answer]
No, sorry, that's currently not implemented. You can check out ergast.com, though. It's one of the sources where FastF1 pulls data from. The data is available there, so you could use that API directly, to get the data.

**Reply by danielsaed** [reply]
Thank you! I'll have a look at that
